### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by anuschka**
Addressed issue #75 and included login/logout option in the navigation.

### CODE REVIEW

File: web/templates/layout/navigation.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	<ul class="nav navbar-nav navbar-right">
+        <li><a href="{% url 'pages-about' %}">About</a></li>
+        <li><a href="{% url 'pages-events' %}">Events</a></li>
+        <li><a href="{% url 'pages-links' %}">Links</a></li>
+	</ul>
+    </div>
+	<div class="collapse navbar-collapse" id="myNavbar">
+		<ul class="nav navbar-nav" role="menu">
+		<li>
+			<a href="{% url 'pages-index' %} "first" id="index">
```
Comment (goranche) [inline web/templates/layout/navigation.html]: what's with the "first" in this a tag?

**Comment by goranche** [comment]
fix indentation

**Comment by goranche** [comment]
@anuschka is the "Sign in" option supposed to change to "Sign out" when signed in?

File: web/templates/layout/navigation.html (hunk 1 above)
Comment (anuschka) [inline web/templates/layout/navigation.html]: It is a typo.

**Comment by anuschka** [comment]
@goranche You are totally right and I am trying to figure out why it does not change to "Hi, name" with a drop down to sign out on the index.html page but it does on the other pages.

**Comment by anuschka** [comment]
Found the bugger :smile_cat: in views.py. 
>def index(request):
>return render_to_response('pages/index.html',{},context_instance=RequestContext(request))

**Comment by fbbunny** [comment]
Looks good to me! We just need to do something with the front picture (and this does not refer to the exact picture groupmeet.jpg, that's just and example we all know needs to be changes :))

**Comment by PettraV** [comment]
no comments, just congrats :+1:   :)

File: web/templates/layout/navigation.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	<div class="collapse navbar-collapse" id="myNavbar-top-right">
+		<ul class="nav navbar-nav navbar-left">
+			<li><a class="navbar-brand" href="{% url 'pages-index' %}">TrackCat</a></li>
+		</ul>  
+		<ul class="nav navbar-nav navbar-right">
+			<li><a href="{% url 'pages-about' %}">About</a></li>
+			<li><a href="{% url 'pages-events' %}">Events</a></li>
+			<li><a href="{% url 'pages-links' %}">Links</a></li>
+		</ul>
+		</div>
```
Comment (ialja) [inline web/templates/layout/navigation.html]: Indentation.

**Comment by ialja** [comment]
Might be just me, but I'm having issues reading the links (very poor contrast). Also, why are they scattered all over the place, randomly (mis)aligned? The login/logout now works, but why is in a different color than other links?

screen shot 2015-02-16 at 10 51 12

**Comment by ercchy** [comment]
Sorry, wrong button! :)

**Comment by ialja** [comment]
@anuschka You cannot make design changes that break the rest of the page (Catz lost their heads in the pic due to the change, the big TrackCat header is now floating in mid-air :disappointed:). Well, you can break stuff this way, but there's no way we're merging that :wink: 

Also, we're not happy with you copying the positioning of the items from the Code Week website. That positioning had its reasons that cannot be transferred to TrackCat. If you want to improve the navigation, start by thinking what's needed where, and design a layout that makes sense for your project. 

Closing this PR, I'd suggest you start fresh with a layout and design that's made for TrackCat.

screen shot 2015-02-17 at 10 18 39

**Comment by anuschka** [comment]
It was a good learning experience :books:

**Pull request closed without merging**